rustls/src/server/server_conn.rs

```diff
@@ -730,25 +730,36 @@ impl Acceptor {
     /// Returns `Ok(Some(accepted))` if the connection has been accepted. Call
     /// `accepted.into_connection()` to continue. Do not call this function again.
     ///
-    /// Returns `Err(err)` if an error occurred. Do not call this function again.
-    pub fn accept(&mut self) -> Result<Option<Accepted>, Error> {
+    /// Returns `Err((err, alert))` if an error occurred. If an alert is returned, the
+    /// application should call `alert.write()` to send the alert to the client. Do not call
+    /// this function again.
```

Comment:
I think it might be worth clarifying here if "this function" is `accept` or `alert.write`.

Reply:
Fair, clarified.